src/js/main.js

```diff
@@ -366,6 +368,61 @@ Site.Earth = {
   init: function() {
     var _this = this;
 
+    if (_this.canHandleThree()) {
+      _this.initThreeEarth();
+    } else {
+      _this.initFlatEarth();
+    }
+
+  },
+
+  canHandleThree: function() {
+    var _this = this;
+
+    // init MobileDetect
+    var md = new MobileDetect(window.navigator.userAgent);
+
+    if (md.is('iOS') || md.is('Safari')) {
```

Comment:
Seems like not targeted at desktop anything: https://github.com/hgoebl/mobile-detect.js/issues/3#issuecomment-36220433